src/modules/listClass.js

```diff
@@ -0,0 +1,6 @@
+export default class NewItem {
+  constructor(task) {
+    this.task = task,
+    this.complete = false,
+    this.index = 0;}
```

Comment:
Hi @daudi13, from the requirement of this project, the property index should be set to the value of the new array length ie `taskArr.length`. If the 4th task is added to the list(taskArr), the index of the task should be set to 5. In your case, if the 4th task is added to the list the index is set to 3 ie index: 3, since you have initialized the index value to zero. Please let's fix it and get closer to approval.